cli/src/seth_cli/client/client.go

```diff
@@ -83,7 +83,10 @@ func (c *Client) Get(address []byte) (*EvmEntry, error) {
 	}
 
 	var data string
-	if body.Data != nil {
+	if body.Data == nil {
+		err := fmt.Errorf("please add option --nonce=0")
```

Comment:
This error message is a bit too specific for this method. There is a chance the body.Data could be nil without encountering the nonce bug